### DISCUSSION TITLE
Renault Kadjar (RLink 2) black screen after hardware failure

### DISCUSSION THREAD

**Discussion by pgale** [cedricp/ddt4all#992] [Q&A]
Hi,

I know there are some very talented people here that might be able to point me in the right direction. This will be a long-ish post but I wanted to get everything down so I can hopefully get some pointers as to what to do next.

**Car and kit:**
Renault Kadjar Signature Nav TCe 130 (RLink 2) - UK right-hand drive
Built around Feb 2019
I think the vehicle type might be HLMR but not 100% sure
Other details from the delivery slips - Model FEH052 Version FEHBOS MRE6R M2
RLink 2 unit - Original model no: LANR14, part No: 28115 (1779n written next to it), HW ver: 1.31.2.3 SW ver: 2.2.15 (703 written)
Original USB module: 280239732R  DQ15881510
Screen module: A14677 14/12/15 HW;0E.01  SW;05.04 

I have DDT4ALL and database from July 2020
OBDLink SX USB OBD cable (unmodified)

**What happened**
About a year ago, the car was being driven and smoke started coming out of the USB module. Shortly after (2 minutes), the screen went black/dead. The radio is still working and can be controlled by the steering wheel controls and station displays in the instrument cluster screen. The power button and rotary volume selector knob does not work.

**What I have tried**
I've only just got around to looking at it as the Renault garage quoted £1500+ to repair the system. Thinking it was the radio nav unit and USB module at fault, I bought an updated replacement radio nav unit and USB module from an established/trusted seller on eBay and the unit has been fully coded to my car and VIN no. These new units are the latest versions and also support carplay etc. I also bought a replacement second-hand screen from a scrap car (same part number etc). However, when the new units are fitted, the screen is still dead and the power button/volume control does not operate. The radio still works as before and actually the DAB radio reception is massively improved as part of the fix in the new radio version - so at least I've gained something! I've removed the radio unit, screen and USB module to fault find and determined that the USB module has indeed been fried. The rear metal housing from one of the USB connectors was loose in the unit. Putting 2+2 together, it looks like the loose metal piece has caused a short, and burnt-out components - see pics:

![IMG_0308](https://github.com/cedricp/ddt4all/assets/513044/19e81cfa-992e-42db-9fa9-0b15972c893d)
![IMG_0309](https://github.com/cedricp/ddt4all/assets/513044/50d16298-a726-49ab-ab0b-0668e9d740d5)
![IMG_0310](https://github.com/cedricp/ddt4all/assets/513044/4f56d3f7-118d-477c-b3bb-33122d9409a1)

I've checked the two radio labelled fuses in the glovebox and they seem ok. I've used DDT4ALL to have a quick snoop around but I think there could still be a hardware fault or other blown fuse somewhere?

On the buttons next to the screen (parking assist, lane guidance and home button), the parking assist button does light up so at least part of that is working. I've also tried holding down the power button for a long time - multiple attempts for 20-30-60 seconds but it doesn't reset the radio as some others have tried before.

**Questions**

1. What fuses may have been affected and are there any other fuses or voltage regulators etc that I should check?
2. Is the screen powered from the radio nav unit or directly from somewhere else (12V or something else?)
3. Is the USB module powered directly from the radio nav unit or from somewhere else? I can see all three units are connected together with USB cables but they also have 8 pin connectors (are these CAN or something else?)
4. Is there any other unit that could have been damaged or has set itself to disable the screen and can I check this with DDT4ALL?
5. Should the screen be powered if the radio is actually working?
6. What car model should I use in DDT4ALL as there are several Kadjar entries?
7. To view the RLink 2 data, do I need to modify the OBD cables/connections as I've seen for RLink 1?

I'm at a bit of a loss now as to what to do/check next. My thoughts are that having replaced the radio, USB and screen that it should work unless there's another component at fault.

Any thoughts?

Many thanks!

**Comment by mgebka-wq** [comment]
@pgale I have the exact same problem with the R-Link 2 system. Were you able to determine the cause of the issue? I replaced the radio and the USB, but it didn’t help.

**Comment by pgale** [comment]
No, unfortunately not yet. Still dead. I was going to try a new USB cable between the screen and unit but not got around to it yet.

________________________________

**Reply by DerKaym** [reply]
@pgale Hey i am a Diagnostic Tech from Renault. If you want to try to find the issue with ur Rlink 2 we can check it together.

If want any Info/Wire Diagrams etc just hit me up on Discord: der_kaym

Regards
Kaym

**Reply by pgale** [reply]
Thanks - I did find some diagrams a while ago (paid and downloaded from the Renault system). I couldn't see any other modules or fuses etc that would obviously be causing the problem. My only other option was to try a standard USB cable between the USB module and R-Link 2 unit to see if somehow the USB cable has been damaged (I think that pretty unlikely though). Still at a loss as to what else could be wrong. Any ideas?

**Reply by DerKaym** [reply]
Hey Sorry for the late response, did you try to Update the Rlink?

There is a know Problem with the cooling Fan routin after an Update that shorts out the Rlink.
